packages/webhooks/tsconfig.json

```diff
@@ -13,6 +13,9 @@
         },
         {
             "path": "../types"
+        },
+        {
+            "path": "../shared"
```

Comment:
I am not sure we want to bring shared as a dependency of webhook.
Would it make sense to move the redactHeaders function into `utils`. It doesn't have any dependency and no business domain logic. I think it would be a good candidate to live in `utils`

Reply:
agreed 👍🏻

Reply:
Fixed in https://github.com/NangoHQ/nango/pull/3321